cluster-autoscaler/core/scale_down.go

```diff
@@ -431,9 +434,9 @@ func drainNode(node *apiv1.Node, pods []*apiv1.Pod, client kube_client.Interface
 		return fmt.Errorf("Failed to drain node %s/%s, due to following errors: %v", node.Namespace, node.Name, evictionErrs)
 	}
 
-	// Evictions created successfully, wait maxGratefulTerminationSec to see if nodes really disappeared
+	// Evictions created successfully, wait maxGratefulTerminationSec + DeletionSafetyMarginto to see if pods really disappeared.
```

Comment:
Can we rename this to `maxGracefulTerminationSec` in this PR please? I'm not sure if changing the flag name is appropriate for minor releases since it'll break some people's deployment but we should at least update the var name and leave a note in the flag description to update it to `--max-graceful-termination-sec`